cmd/sriovdp/sriov-device-plugin.go

```diff
@@ -515,6 +518,17 @@ func (sm *sriovManager) Allocate(ctx context.Context, rqt *pluginapi.AllocateReq
 		envmap["SRIOV-VF-PCI-ADDR"] = pciAddrs
 
 		containerResp.Envs = envmap
+
+		if isRdmaDirExists {
```

Comment:
Is there a solid way to identify whether a certain device is rdma capable ?
I don't think we want to mount rdma device path to every pod even if device allocated to pod doesn't support rdma.
for example, if there are two types of device on the same host, one is rdma capable, the other is not, then we should avoid mounting rdma path to the other device.

Reply:
@zshi-redhat I updated the code to check if the device is supporting rdma by checking the full path of  ` /sys/class/pci_bus/<pciBus>/device/<pciAddress>/infiniband`
e.g. ` /sys/class/pci_bus/0000\:02/device/0000:02:00.4/infiniband`